### PR TITLE
Return commit so that subsequent workflow steps could use it.

### PR DESCRIPTION
**Pull request by DebbyLee**
To solve the issue I request in https://github.com/juliangruber/merge-pull-request-action/issues/27

1. Add the readme to show case what is commit
2. Update action.yml
3. Update the index.js to set output

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +12,15 @@ steps:
       github-token: ${{ secrets.GITHUB_TOKEN }}
       number: 1
       method: squash # merge rebase
-      repo: juliangruber/octokit-action
+      repo: juliangruber/octokit-action
```
Comment (juliangruber) [inline README.md]:
```suggestion
      repo: juliangruber/octokit-action
```

File: action.yml [hunk 2]
```diff
@@ -18,6 +18,9 @@ inputs:
   repo:
     description: 'Pull Request repo in owner/repo format'
     required: false
+outputs:
+  commit:
+    description: 'commit sha'
```
Comment (juliangruber) [inline action.yml]:
```suggestion
    description: 'Commit sha of merge commit'
```

File: index.js [hunk 3]
```diff
@@ -19,11 +19,13 @@ const main = async () => {
 
   const octokit = new GitHub(token)
 
-  await octokit.pulls.merge({
+  let response = await octokit.pulls.merge({
```
Comment (juliangruber) [inline index.js]:
```suggestion
  const response = await octokit.pulls.merge({
```

File: index.js [hunk 4]
```diff
@@ -19,11 +19,13 @@ const main = async () => {
...
     ...repoObject,
     pull_number: number,
     merge_method: method
   })
+
+  core.setOutput("commit", response.data.sha);
```
Comment (juliangruber) [inline index.js]:
```suggestion
  core.setOutput('commit', response.data.sha)
```

File: node_modules/.package-lock.json
Comment (juliangruber) [inline node_modules/.package-lock.json:1]: could you please revert the changes to `node_modules/`?

**Review by juliangruber — CHANGES_REQUESTED** [review]

**Comment by DebbyLee** [comment]
thanks for the review. addressed the comment

**Comment by juliangruber** [comment]
Thank you!

https://github.com/juliangruber/merge-pull-request-action/releases/tag/v1.2.0
